src/components/navbar.jsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import AppBar from '@mui/material/AppBar';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
+
+function ResponsiveAppBar() {
```

Comment:
Let's name the component and the file the same thing for better code organization and navigation. Here are come other interesting naming conventions: https://github.com/airbnb/javascript/tree/master/react#naming